src/pages/tasks/TaskAssigneeSelectorModal.js

```diff
@@ -183,7 +182,7 @@ function TaskAssigneeSelectorModal(props) {
                 <>
                     <HeaderWithBackButton
                         title={props.translate('task.assignee')}
-                        onBackButtonPress={() => (lodashGet(props.task.report, 'isExistingTaskReport') ? Navigation.dismissModal() : Navigation.goBack(ROUTES.NEW_TASK))}
+                        onBackButtonPress={() => (props.route.params && props.route.params.reportID ? Navigation.dismissModal() : Navigation.goBack(ROUTES.NEW_TASK))}
```

Comment:
```suggestion
                        onBackButtonPress={() => (lodashGet(props.route.params, 'reportID') ? Navigation.dismissModal() : Navigation.goBack(ROUTES.NEW_TASK))}
```

Can we use `lodashGet` here instead?

Reply:
Yes, updated!